web/README.md

```diff
@@ -35,6 +35,26 @@ running Agama server instance. This is especially useful if you use the Live ISO
 which does not contain any development tools, you can develop the web frontend
 easily from your workstation.
 
+Example of running from different machine:
+
+```
+  # backend machine
+  # using ip of machine instead of localhost is important to be network accessible
+  agama-web-server serve --address 10.100.1.1:3030
```

Comment:
```suggestion
  agama-web-server serve --address 10.100.1.1:3000
```

Reply:
and that reminds me that I should update that documentation now with ssl enabled